packages/design/src/ResponsiveBreakpoints.mdx

```diff
@@ -0,0 +1,48 @@
+---
+name: Breakpoints
+menu: Design
+route: /breakpoints
+---
+
+import { Button } from "@jobber/components/Button";
+
+# Responsive Breakpoints
+
+To make most components responsive, you will want to use the
+[useResizeObserver hook](/hooks/use-resize-observer) so that the component can
```

Comment:
I wonder if we should make this a bit more prominent some how? As it is an important point that is quite different from how we are used to doing responsiveness at Jobber.

Reply:
@seanhealy I had thought of this in another PR, but pulled it in here. I updated the atlantis menu to bring design and hooks up to the top of the menu to make them more prominent. I think that will help users get their eyes on it. 

Thoughts?